release notes/v0.7.0.md

```diff
@@ -0,0 +1,51 @@
+xk6-browser v0.7.0 is here! :tada:
+
+We've been busy preparing for big changes in the future, so this minor release is smaller than usual. It contains a new feature to unblock users using `dialog` boxes, and internal improvements.
+
+
+## New features
+
+- `xk6-browser` will now automatically dismiss dialog boxes (`alert`, `confirm`, `prompt`, and `beforeunload`). ([#663](https://github.com/grafana/xk6-browser/pull/663))
+
+
+## Improvements
+
+- You can now launch a new browser without defining an empty options object. ([#649](https://github.com/grafana/xk6-browser/pull/649))
+    
+    With the empty object:
+
+    ```js
+    const browser = chromium.launch({});
+    ```
+
+    Without the empty object:
+
+    ```js
+    const browser = chromium.launch();
+    ```
+
+
+### Documentation
+
+- Updated the video to match what we released in `v0.6.0`, and it's still relevant for `v0.7.0`. ([#639](https://github.com/grafana/xk6-browser/pull/639))
+
+- Simplified the README. You will now be redirected to the [documentation](https://k6.io/docs/javascript-api/xk6-browser/). ([#656](https://github.com/grafana/xk6-browser/pull/656))
+
+- The project roadmap has been updated. ([#658](https://github.com/grafana/xk6-browser/pull/658))
```

Comment:
I made some improvements as we discussed, and rearranged some of the items.

Feel free to commit this and then make your own changes :)

```suggestion
## Improvements

- Improved launching of a browser without providing any options. ([#649](https://github.com/grafana/xk6-browser/pull/649))
 
  We couldn't launch a new browser without providing an empty object. We've fixed this, so now you can launch a new browser without an empty object. 

    With the empty object:

    ```js
    const browser = chromium.launch({});
    ```

    Without the empty object:

    ```js
    const browser = chromium.launch();
    ```

### Documentation

- Updated the demo video on our README. ([#639](https://github.com/grafana/xk6-browser/pull/639))

  Now it shows the usage for xk6-browser version v0.6.0 and above.

- Simplified the README. ([#656](https://github.com/grafana/xk6-browser/pull/656))

  You can now find the documentation [here](https://k6.io/docs/javascript-api/xk6-browser/).

- The project roadmap has been updated. ([#658](https://github.com/grafana/xk6-browser/pull/658))
```